src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -206,7 +216,7 @@ export const PositionsTable = ({ positions, displayLine = false }: PositionsProp
     }
 
     // If user is lender and position status is PROPOSED, return revoke consent action
-    if (getAddress(position.lender) === userWallet && position.status === 'PROPOSED') {
+    if (getAddress(position.lender) === userWallet && position.status === PROPOSED_STATUS) {
```

Comment:
this code block is actually dead code with the way `position.propsosal` is being handled in `getUserProposalActions`.

Just deleted all the logic since it's dead code